coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/view/walkthroughmessage/WalkthroughMessageView.kt

```diff
@@ -18,18 +23,50 @@ class WalkthroughMessageView @JvmOverloads constructor(
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
 
     private var position: WalkthroughMessagePosition = WalkthroughMessagePosition.BELOW
+    private var listener: WalkthroughScrollessMessageView.WalkthroughButtonClicklistener? = null
```

Comment:
Don't use generic names for variables
```suggestion
    private var walkthroughButtonClicklistener: WalkthroughScrollessMessageView.WalkthroughButtonClicklistener? = null
```